### ISSUE TITLE
Version 0.5.0 doesn't get properly installed via NPM

### ISSUE THREAD

**Issue by nowherenone** [Hargne/jest-html-reporter#11]
Now right after running "npm install", the ./node_modules/jest-html-reporter folder doesn't have /src and /test folders at all:
```
07/13/2017  10:39 PM    <DIR>          coverage
07/13/2017  10:39 PM                10 .npmignore
07/13/2017  10:39 PM             3,219 CODE_OF_CONDUCT.md
07/13/2017  10:39 PM               218 CONTRIBUTING.md
07/13/2017  10:39 PM               484 jest.config.json
07/13/2017  10:39 PM             1,069 LICENSE
07/13/2017  10:39 PM             5,032 package.json
07/13/2017  10:39 PM             2,178 README.md
07/13/2017  10:39 PM             2,822 test-report.html
```
But it contains a /coverage folder instead.    Version 0.4.1 installs fine.
npm v.4.6.1 , node v.7.9.0

**Comment by Hargne** [comment]
Thank you for noticing this issue. I will have a look as soon as possible.

**Comment by Hargne** [comment]
@nowherenone I have now published `v0.5.1` in which the issue now should be resolved. Would you mind cross-checking that it works as intended?

Thank you!

**Comment by nowherenone** [comment]
It works fine now, thank you so much for a quick response!
